server/api/crud/alerts.py

```diff
@@ -170,9 +169,10 @@ def process_event(
 
                 if state_obj is None:
                     state_obj = {"events": [event_data.timestamp]}
+                else:
+                    state_obj["events"].append(event_data.timestamp)
```

Comment:
Can be improved a bit
```suggestion
                state_obj = self._states.get(alert.id, None) or {}
                state_obj.setdefault("events", [])
                state_obj["events"].append(event_data.timestamp)
```